instrumentation/google.golang.org/grpc/otelgrpc/example/server/main.go

```diff
@@ -88,7 +89,7 @@ func (s *server) SayHelloBidiStream(stream api.HelloService_SayHelloBidiStreamSe
 	for {
 		in, err := stream.Recv()
 
-		if err == io.EOF {
+		if errors.Is(err, io.EOF) {
```

Comment:
Have you checked the performance overhead? While I have nothing against using `errors.Is` and `errors.As` in business logic I am a uncertain if we to use it in instrumentation libraires. As far as I remember it uses some kind of reflection (`internal/reflectlight`).

Reply:
Related: https://github.com/open-telemetry/opentelemetry-go/issues/5551

Reply:
Does not look like an issue.